client/mm/libxc/binance.go

```diff
@@ -1831,10 +1869,9 @@ func getDEXAssetIDs(binanceSymbol string, tokenIDs map[string][]uint32) []uint32
 		if isRegistered(assetID) {
 			assetIDs = append(assetIDs, assetID)
 		}
-
 	}
 
-	if tokenIDs, found := tokenIDs[dexSymbol]; found {
+	if tokenIDs, found := tokenIDs[coin]; found {
```

Comment:
```suggestion
	if tokenIDs, found := tokenIDs[strings.ToUpper(coin)]; found {
```

This fixes the bug we discussed in matrix.